packages/x-charts/src/context/CartesianProvider/computeValue.ts

```diff
@@ -64,21 +74,21 @@ export function computeValue(
       formattedSeries,
     );
 
-    const range = axis.reverse
-      ? [drawingArea.left + drawingArea.width, drawingArea.left]
-      : [drawingArea.left, drawingArea.left + drawingArea.width];
+    const range = getRange(drawingArea, axisName, axis.reverse);
 
     if (isBandScaleConfig(axis)) {
       const DEFAULT_CATEGORY_GAP_RATIO = 0.2;
       const DEFAULT_BAR_GAP_RATIO = 0.1;
 
       const categoryGapRatio = axis.categoryGapRatio ?? DEFAULT_CATEGORY_GAP_RATIO;
-      const barGapRatio = axis.barGapRatio ?? DEFAULT_BAR_GAP_RATIO;
+      const barGapRatio = axisName === 'x' ? axis.barGapRatio ?? DEFAULT_BAR_GAP_RATIO : 0;
```

Comment:
```suggestion
      const barGapRatio = axis.barGapRatio ?? DEFAULT_BAR_GAP_RATIO;
```

This looks like a bug fix. The band scale should not have different defautl depending on the x/y axis